examples/templates/devcontainer-kubernetes/main.tf

```diff
@@ -222,7 +257,7 @@ resource "kubernetes_deployment" "main" {
 
         container {
           name              = "dev"
-          image             = local.devcontainer_builder_image
+          image             = var.cache_repo == "" ? local.devcontainer_builder_image : envbuilder_cached_image.cached.0.image
           image_pull_policy = "Always"
           security_context {}
           env {
```

Comment:
review: leaving as-is until https://github.com/coder/terraform-provider-envbuilder/issues/31 is resolved.

There may be some additional Terraform jiggery pokery requried to convert `local.envbuilder_env` into a block list.

Reply:
A for-loop would be nice here to set all the envs, wdyt?

Right now it can come from `locals.envbuilder_env`, but eventually from `envbuilder_cached_image.cached.0.env`.

```tf
env {
	for_each = locals.envbuilder_env
	name = each.key
	value = each.value
}
```

Not sure if `tomap` is required (`for_each = tomap(locals.envbuilder_env)`), probably not?

Reply:
Yeah, but then that needs to be referenced in the `kubernetes_pod` spec below. I think that may need a `dynamic{}` block?

Reply:
Possibly, my tf foo is not strong enough 😄

Reply:
I'll address this in a follow-up. The current way the envbuilder provider outputs the computed env isn't really conducive to this scenario right now anyway.